pybossa/view/projects.py

```diff
@@ -362,7 +362,7 @@ def new():
     # Sort list of subproducts (value) for each product (key).
     prodsubprods = {key:sorted(value) for key, value in current_app.config.get('PRODUCTS_SUBPRODUCTS', {}).items()}
     data_classes = [(data_class, data_class, {} if enabled else dict(disabled='disabled'))
-        for data_class, enabled in current_app.config.get('DATA_CLASSIFICATION', ('', False))
+        for data_class, enabled in current_app.config.get('DATA_CLASSIFICATION', [('', False)])
```

Comment:
This looks good to me. I'm just thinking it would be nice to have a unit test that would have caught this issue; or a new unit test to ensure this is working as expected. Otherwise, looks good, thank you.